spec/system/user_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'rails_helper'
+
+describe 'User index page', type: :feature do
```

Comment:
- [ ] **[OPTIONAL]** You have added the spec tests for the user index and show pages in this one file. I would recommend you split them into two files for better code organization.